### ISSUE TITLE
c++/winrt Windows.UI.Input.Preview.Injection seems broken

### ISSUE THREAD

**Issue by Alkeryn** [Microsoft/xlang#382]
So there are many issues
But here are some of them
i can't create an InputInjector object
If i write
`InputInjector  foo;`  the compiler tells me that it has no constructor
If i try to create an InjectedInputPenInfo object and assign one of its variable i get compiler errors ex:
```cpp
InjectedInputPenInfo foo;
foo.Pressure = 0.5;
```
it tells me `m_header is uninitialized`
Also for some reason i got foo.Pressure.Pressure autocompletion
like if foo.Pressure contained foo.* which is definitely weird behavior

And lastly there are those weird classes such as 
IInputInjector
or IInjectedInputPenInfo
basically the same name but with an "I" before on top

And to end, it seems to work with c#/uwp but i don't want to use C# for that project nor it to be an uwp only app

Or maybe i should make derivation for those classes ?

**Comment by Alkeryn** [comment]
might have been a documentation issue or missunderstanding, after toying around
this seems to work : 
![image](https://user-images.githubusercontent.com/11599075/57114596-3ffc0800-6d4a-11e9-8201-adf648e11f99.png)

**Comment by Alkeryn** [comment]
Well i just can't get it to work, i think i'll just write a driver and be happy

**Comment by kennykerr** [comment]
Your best bet for questions about specific APIs is to ask a question on Stack Overflow, ideally including a small self-contained repro.

**Comment by Alkeryn** [comment]
Yeah well i ended up writting a driver because i couldn't get that part of the api to work
i think there is an issue with the api itself or maybe you are supposed to use it diferently but there is literally 0 documentation about it and it dosen't behave like the documentation for the c# way of doing it at all
so i just gave up on it and a driver is more efficient anyway
